app/views/conferences/_conference_details.html.haml

```diff
@@ -27,8 +27,8 @@
               - if conference.user_registered?(current_user)
                 = link_to "My Registration", conference_conference_registration_path(conference.short_title), class: 'btn btn-default'
               - else
-                = link_to "Register", new_conference_conference_registration_path(conference.short_title), class: "btn btn-default", disabled: conference.registration_limit_exceeded?
-                - if conference.registration_limit_exceeded?
+                = link_to "Register", new_conference_conference_registration_path(conference.short_title), class: "btn btn-default", disabled: cannot?(:new, Registration.new(conference_id: conference.id))
+                - if cannot?(:new, Registration.new(conference_id: conference.id)) && conference.registration_limit_exceeded?
```

Comment:
Because we want the message "Sorry, no places left" to appear only when the reason the user doesn't have the ability to register is that the registration limit had been exceeded